lib/hotel-page/index.js

```diff
@@ -129,12 +129,11 @@ class HotelPage extends Component {
               </div>
               {this.renderFactlist(this.retrieveAmenities())}
             </div>
-            <div className='bookButtonContainer'>
+            <div className='bookButtonContainer clearfix'>
               <div className='ppp'>ppp <span className='hotelPrice'>
-                {packageOffer.price.perPerson},-
+                {parseInt(packageOffer.price.perPerson, 10).toLocaleString('da-DK')},-
```

Comment:
Please add an issue to the backlog to create a test for this. 
merging now to get this in.